src/index.js

```diff
@@ -0,0 +1,32 @@
+import './style.css';
+// arrow todolistect
+const todolist = [
+  {
+    discription: 'js',
+    completed: '',
+    index: 1,
+  },
+  {
+    discription: 'java',
+    completed: '',
+    index: 2,
+  },
+  {
+    discription: 'html',
+    completed: '',
+    index: 3,
+  },
+];
+// display
+const list = document.querySelector('.todolist');
```

Comment:
- [ ] Your todo lists are not loaded on your page because the `todolist` class name element you're querying for  doesn't exist in your html file. What exists in the html file is a class name of `todos`. Please kindly query for the right class name.

<img width="609" alt="image" src="https://user-images.githubusercontent.com/52934128/222903756-58571ac2-b569-4a2d-a4a8-ce3b9b1e0a29.png">